skaha/src/main/java/org/opencadc/skaha/SkahaAction.java

```diff
@@ -111,6 +115,7 @@ public abstract class SkahaAction extends RestAction {
     public static final String SESSION_TYPE_CONTRIB = "contributed";
     public static final String SESSION_TYPE_HEADLESS = "headless";
     public static final String TYPE_DESKTOP_APP = "desktop-app";
+    public static final String TRUE = "true";
```

Comment:
Is this being used?  If so, I think Boolean.TRUE.toString() (or something like that) is the same.